gammapy/time/tests/test_lightcurve.py

```diff
@@ -188,23 +195,136 @@ def test_lightcurve_estimator_spectrum_datasets():
     assert_allclose(lightcurve.table["norm_errp"], [0.044231, 0.04377], rtol=1e-5)
     assert_allclose(lightcurve.table["norm_errn"], [0.04374, 0.043226], rtol=1e-5)
     assert_allclose(lightcurve.table["norm_ul"], [1.077213, 1.036237], rtol=1e-5)
-    assert_allclose(lightcurve.table["sqrt_ts"], [37.16802, 37.168033], rtol=1e-5)
-    assert_allclose(lightcurve.table["ts"], [1381.461738, 1381.462675], rtol=1e-5)
+    assert_allclose(lightcurve.table["sqrt_ts"], [26.773925, 25.796426], rtol=1e-5)
+    assert_allclose(lightcurve.table["ts"], [716.843084, 665.455601], rtol=1e-5)
     assert_allclose(lightcurve.table[0]["norm_scan"], [0.2, 1.0, 5.0])
     assert_allclose(
         lightcurve.table[0]["stat_scan"],
         [444.426957, 23.375417, 3945.382802],
         rtol=1e-5,
     )
+    assert len(estimator.group_table_info) == 2
+    assert estimator.group_table_info["Name"][0] == "dataset_1"
+    assert_allclose(estimator.group_table_info["Tstart"], [55197.0, 55197.04166666667])
```

Comment:
Make a separate test function for the `group_datasets_in_time_interval`. This will be useful if this moves on `Datasets` for instance.